### PR TITLE
Update padigitalreqd

### PR DESCRIPTION
**Pull request by lfinnigan**
Added pattern id to check that edm:isShownAt is a URL. Updated xspec: added dataProvider element throughout since that was added to the validation; removed checks for thumbnails since those were removed from validation; added check that isShownAt is URL; added checks for dataProvider

### CODE REVIEW

File: validations/padigital_reqd_fields.xspec [hunk 1]
```diff
@@ -444,109 +456,150 @@
...
+        <edm:isShownAt>This is not a URL</edm:isShownAt>
+        <edm:preview>http://digital.library.temple.edu/cdm/ref/collection/p16002coll25/id/0/tnail</edm:preview>
+        <edm:dataProvider>Temple University</edm:dataProvider>
+        <edm:provider>PA Digital</edm:provider>
+      </oai_dc:dc>
+    </x:context>
+    <x:expect-assert id="ItemURL2"/>
+  </x:scenario>
+</x:scenario>
+</x:scenario>
```
Comment (htomren) [inline validations/padigital_reqd_fields.xspec]: you have three closing <x:scenario> but there should only be two

File: validations/padigital_reqd_fields.xspec [hunk 2]
```diff
@@ -444,109 +456,150 @@
     </x:scenario>
   </x:scenario>
 
-  <x:scenario label="ThumbnailURLElementPattern">
-    <x:scenario label="valid">
+  <x:scenario label="not valid: isShownAt URL is not URL">
```
[the remark this answers is not in the document]
Comment (htomren) [reply validations/padigital_reqd_fields.xspec]: Never mind. Instead, perhaps delete the closing </x:scenario> on the previous line so stay within the same scenario for ItemURLElementPattern

File: validations/padigital_reqd_fields.xspec (hunk 1 above)
Comment (htomren) [reply validations/padigital_reqd_fields.xspec]: This might be OK after all but check the nesting of the x:scenarios to be sure...

Comment (lfinnigan) [reply validations/padigital_reqd_fields.xspec]: will submit updated in another pr

**Pull request closed without merging**